typings/helpers/Constants.d.ts

```diff
@@ -38,4 +41,7 @@ export namespace Constants {
   export function removeDimensionsEventListener(
     callback: (event: { window: ScaledSize, screen: ScaledSize }) => void
   ): void;
+
+  export function getTabletMargins(): number; // TODO: remove
+
```

Comment:
this is also private. shouldn't be exposed. 
it's not even implemented here and will result in an error if someone will try to use it